python-bindings/Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY:
+
+all:
+	go build -o libcarrow_bindings.so -buildmode=c-shared carrow_bindings.go && \
+#	pip install -r requirements.txt && \
+	python setup.py build_ext --inplace
+
+clean:
+	rm -rf *.so *.cpp *.c *.h build
+
+test:
+	ldconfig && python runner.py
```

Comment:
IMO the `ldconfig` should be in the `Dockerfile`

Reply:
is 	LD_LIBRARY_PATH=$${PWD} python runner.py OK?